README.md

```diff
@@ -76,7 +76,16 @@ Use Snowflake CLI to build a function or stored procedure, or create a streamlit
 
 ### Prerequisites
 
-You must add your credentials to connect to Snowflake before you can use Snowflake CLI. You can add your Snowflake credentials using any of the following methods.
+You should add your credentials to connect to Snowflake before you can use Snowflake CLI. You can add your Snowflake credentials using any of the following methods.
+
+#### Temporary connection
+
+If you don`t want to store your credentials, or you are performing one-time operation on an account, you won't use frequently, you can provide all necessary data, using command
+line flags and `--temporary-connection` option:
+```commandline
+$ snow snowpark warehouse status --temporary-connection --acount "my_account" --user "my_username" --password "my_password"
+```
+
```

Comment:
Let's not add here more info. We were supposed to remove the info altogether https://github.com/Snowflake-Labs/snowcli/pull/287

Reply:
Ok, but, as we have information on possible ways of connection - from config, env variables etc. So shouldn`t we add a little note about --temporary-connection

Reply:
This note should be removed I think